app/Models/Entry.php

```diff
@@ -678,11 +698,21 @@ public function matches(FreshRSS_BooleanSearch $booleanSearch): bool {
 						$ok &= stripos($this->link, $url) !== false;
 					}
 				}
+				if ($ok && $filter->getIntitleRegex()) {
```

Comment:
I think you meant this to be `getInurlRegex`

Reply:
Well spotted!